src/phel/http.phel

```diff
@@ -144,29 +144,32 @@
       (= method "POST")
       (php/in_array types (php/array "application/x-www-form-urlencoded" "multipart/form-data")))))
 
-
-(defn request-from-globals
-  "Extracts a request from `$_SERVER`"
-  [& [server]]
-  (let [server (or server php/$_SERVER)
-        method (get-method-from-globals server)
-        uri (uri-from-globals server)
-        headers (headers-from-server server)
-        server-protocol (get server "SERVER_PROTOCOL")
+(defn request-from-globals-args
+  "Extracts a request from args"
+  [& [_server _get _post _cookies _files]]
```

Comment:
This line should be like this:
```
[server get-parameter post-parameter cookies files]
```
The `&` is not required because the method should have 5 arguments. I also renamed the `get` argument to `get-parameter` because otherwise this will overwrite the global `get` function in this scope.

Afterwards you have to adopt `_server` to `server`, `_get` to `get-parameter`, ... in the function body.